peps/pep-0778.rst

```diff
@@ -52,10 +52,10 @@ layer of complexity. Not only do they need to implement the above mechanism,
 they must also understand and safely interact with CPython's runtime state,
 including the interpreter loop, garbage collector, thread state, and reference
 counting system. This combination of low-level system manipulation and
-high-level interpreter knowledge makes implementing Python debugging tools
+deep domain specific interpreter knowledge makes implementing Python debugging tools
```

Comment:
"high level" had the wrong connotation. It's still low level knowledge, just in a different domain - low level knowledge of the interpreter, plus low level knowledge of the system it runs within.